src/main/java/de/medizininformatikinitiative/fhir_data_evaluator/PopulationCount.java

```diff
@@ -1,19 +1,23 @@
 package de.medizininformatikinitiative.fhir_data_evaluator;
 
-import org.hl7.fhir.r4.model.CodeableConcept;
 import org.hl7.fhir.r4.model.MeasureReport;
 
 public record PopulationCount(HashableCoding code, int count) {
 
     public MeasureReport.StratifierGroupPopulationComponent toReportStratifierPopulation() {
         return new MeasureReport.StratifierGroupPopulationComponent()
-                .setCode(new CodeableConcept(code.toCoding()))
+                .setCode(code.toCodeableConcept())
                 .setCount(count);
     }
 
     public MeasureReport.MeasureReportGroupPopulationComponent toReportGroupPopulation() {
         return new MeasureReport.MeasureReportGroupPopulationComponent()
-                .setCode(new CodeableConcept(code.toCoding()))
+                .setCode(code.toCodeableConcept())
                 .setCount(count);
     }
+
+    public PopulationCount merge(PopulationCount other) {
+        assert code.equals(other.code);
```

Comment:
If we assume that we only need to handle known populations like the initial-population, it would be a good idea to have a record called InitialPopulation without the code. The advantage will be increased type safty and less runtime overhead by removing the code.